packages/fast-components-react-base/src/auto-suggest/auto-suggest.tsx

```diff
@@ -213,11 +224,13 @@ class AutoSuggest extends Foundation<
         );
     }
 
-    private renderChildren(): React.ReactNode {
+    private renderChildren(alternateChildren?: React.ReactNode): React.ReactNode {
+        const children: React.ReactNode = isNil(alternateChildren)
+            ? this.props.children
+            : alternateChildren;
         if (this.props.filterSuggestions) {
-            const children: React.ReactNode = this.props.children;
             return React.Children.map(
-                children,
+                children as React.ReactNode,
```

Comment:
isn't this var typed as a `React.ReactNode` above? Is this type coercing necessary?

Reply:
Build complains -TS2345: Argument of type '{}' is not assignable to parameter of type 'ReactElement…

Reply:
huh.... okay.